ai_ref_kits/conversational_ai_chatbot/README.md

```diff
@@ -0,0 +1,175 @@
+
+Install libraries and tools:
+
+```shell
+sudo apt install git gcc python3-venv python3-dev
+```
+
+_NOTE: If you are using Windows, you may also need to install [Microsoft Visual C++ Redistributable](https://aka.ms/vs/16/release/vc_redist.x64.exe)._
+
+## Setting up your Environment
+
+### Cloning the Repository
+
+To clone the repository, run the following command:
+
+```shell
+git clone https://github.com/openvinotoolkit/openvino_build_deploy.git
+```
+
+The above will clone the repository into a directory named "openvino_build_deploy" in the current directory. Then, navigate into the directory using the following command:
+
+```shell
+cd openvino_build_deploy/ai_ref_kits/conversational-ai-chatbot
+```
+
+### Creating a Virtual Environment
+
+To create a virtual environment, open your terminal or command prompt and navigate to the directory where you want to create the environment. Then, run the following command:
+
+```shell
+python3 -m venv venv
+```
+This will create a new virtual environment named "venv" in the current directory.
+
+### Activating the Environment
+
+Activate the virtual environment using the following command:
+
+```shell
+source venv/bin/activate   # For Unix-based operating systems such as Linux or macOS
+```
+
+_NOTE: If you are using Windows, use `venv\Scripts\activate` command instead._
+
+This will activate the virtual environment and change your shell's prompt to indicate that you are now working within that environment.
+
+### Installing the Packages
+
+To install the required packages, run the following commands:
+
+```shell
+python -m pip install --upgrade pip 
+pip install -r requirements.txt
+```
+## How to Access LlaMA
+
+_NOTE: If you already have access to the LlaMA model weights, skip to the authentication step, which is mandatory for converting the LlaMA model._
+
+### Accessing Original Weights from Meta AI
+
+To access the original LlaMA model weights:
+
+Visit [Meta AI's website](https://ai.meta.com/resources/models-and-libraries/llama-downloads/) and fill in your details, including your name, email, and organization.
+Accept the terms and submit the form. You will receive an email granting access to download the model weights.
+
+### Using LlaMA with Hugging Face
+
+Set Up a Hugging Face Account: If you don't have one, create a [Hugging Face account](https://huggingface.co/welcome).
+
+Authenticate with Meta AI: Go to the LlaMA model (v2 or v3) page on Hugging Face. To authenticate, enter the same email address you used for the Meta AI website. After authentication, you'll gain access to the model.
+
+To use the model, authenticate using the Hugging Face CLI:
+
+```shell
+huggingface-cli login
+```
```

Comment:
This is no longer needed. They can provide the token when running the script

Reply:
updated the requested changes